wiki/en/Running-a-Server.md

```diff
@@ -81,14 +74,14 @@ This is the default when starting a Server for the first time. Unregistered Serv
 
```

Comment:
Line 73 (new) should just be "Directory".

Reply:
Is this still valid? I thought I changed that.

Reply:
"Directory Server" changed to "Directory".

Reply:
Changed on line 73 (new) but the heading needs changing, too.

Reply:
Ok. Removed all Directory Server occurrences.

Reply:
`Custom Directory` should still be just `Directory`.